package.json

```diff
@@ -15,24 +15,28 @@
   },
   "dependencies": {
     "@aws-sdk/client-accessanalyzer": "^3.716.0",
+    "@aws-sdk/client-application-auto-scaling": "^3.716.0",
     "@aws-sdk/client-auto-scaling": "^3.716.0",
     "@aws-sdk/client-cloudtrail": "^3.716.0",
     "@aws-sdk/client-cloudwatch": "^3.716.0",
     "@aws-sdk/client-cloudwatch-logs": "^3.716.0",
     "@aws-sdk/client-config-service": "^3.716.0",
+    "@aws-sdk/client-dax": "^3.716.0",
```

Comment:
Can you also add the same deps to the cli package.json? it's due to how it's distributed and bundled

Reply:
ok